packages/office-ui-fabric-react/src/components/Icon/ImageIcon.tsx

```diff
@@ -14,17 +14,11 @@ export const ImageIcon: React.FunctionComponent<IImageIconProps> = props => {
   const { className, imageProps } = props;
 
   const nativeProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(props, htmlElementProperties);
-
-  const containerProps = props['aria-label']
-    ? {}
-    : {
-        role: 'presentation',
-        'aria-hidden': imageProps.alt || imageProps['aria-labelledby'] ? false : true
-      };
+  const imageAlt = imageProps['alt'] || props['aria-label'] || '';
```

Comment:
1. When the base Icon is displaying an image, it should follow the same logic for applying `alt` as ImageIcon does (*if* we keep this special logic--see next point; but whatever approach we settle on, it should be the same in both components).
2. I'm not sure that we should automatically transfer `aria-label` to `alt`, since they have some different behaviors (for example, `alt` is used as hover text; possibly other differences too)
3. Can you explain more about the rationale for empty alt text? IIRC this will cause accessibility checkers to flag it.

Reply:
1. Done, image alt added to `Icon.base`, thanks for catching that!
2. the `title` attribute will create hover text, but not `alt`; in the case of an `img` tag, the only difference between `alt` and `aria-label` is that an empty `alt` is used to make an image presentational/hidden, and `alt` is more robust. Both are used to calculate the accessible name of an image, and `aria-label` will actually override `alt` if present. Neither have any effect beyond that.
3. An empty `alt` attribute is the canonical way to mark an image as presentational, and is generally preferred to `aria-hidden`, `role="presentation"`, or other methods (see the [first rule of ARIA](https://w3c.github.io/using-aria/#rule1)). Accessibility checkers won't flag it :)

Reply:
Even though `alt` and `aria-label` have basically the same effect, I'm not sure we should copy `aria-label` to `alt`. Native props like `aria-label` are typically applied exactly as provided, so moving them around feels like violating consumers' expectations.

Also, I don't think we should apply an empty `alt` (hiding the icon) if it has an `aria-labelledby`?